ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -62,14 +64,24 @@ public class NativeViewHierarchyManager
         /// Instantiates the <see cref="NativeViewHierarchyManager"/>.
         /// </summary>
         /// <param name="viewManagers">The view manager registry.</param>
-        public NativeViewHierarchyManager(ViewManagerRegistry viewManagers)
+        /// <param name="dispatcher">The dispatcher of the view.</param>
+        public NativeViewHierarchyManager(ViewManagerRegistry viewManagers,
+#if WINDOWS_UWP
+            CoreDispatcher dispatcher
+#else
+            Dispatcher notUsed
+#endif
```

Comment:
Is this not used because multi-window in WPF uses a single dispatcher, or because this PR makes no attempt at getting multi-window support for WPF?

Reply:
Yeah, I opened two top level windows in a test WPF app and I see them sharing same dispatcher thread. In fact that's the first thread the OS/.Net starts the app with, the simpler model we had before UWP.
That doesn't mean someone cannot create a new dispatcher thread and use that for some of the app windows, but I suspected that was a little too niche.

---
In reply to: [165941520](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r165941520) [](ancestors = 165941520)